packages/amplify-category-analytics/provider-utils/awscloudformation/service-walkthroughs/pinpoint-walkthrough.js

```diff
@@ -3,6 +3,7 @@ const path = require('path');
 const fs = require('fs-extra');
 
 const providerName = 'awscloudformation';
+// FIXME: may be removed from here, since addResource can pass category to addWalkthrough
```

Comment:
Leftover to fix?

Reply:
same to https://github.com/aws-amplify/amplify-cli/pull/2463#discussion_r373736629